### PR TITLE
add explanation for multi value columns + link to config reference wh…

### PR DESCRIPTION
**Pull request by mneedham**
…ere a schema is created

### CODE REVIEW

File: basics/components/schema.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ Data type helps in determining the operations which can be performed on the colu
 `BOOLEAN`, `TIMESTAMP`, `JSON` are added after release `0.7.1`. In release `0.7.1` and older releases, `BOOLEAN` is equivalent to `STRING.`
 {% endhint %}
 
+Pinot also supports columns that contain lists or arrays of items, but there isn't an explicit data type for these columns. Instead, you can indicate that a dimension column accepts multiple values. For more information, see [DimensionFieldSpec](https://docs.pinot.apache.org/configuration-reference/schema#dimensionfieldspec) in the Schema configuration reference.
```
Comment (mayankshriv) [inline basics/components/schema.md:94]: s/explicit data type for these columns/explicit data type to represent these lists or arrays.

**Review by mayankshriv — COMMENTED** [review]
Thanks for adding this. It would be good to also add more information on how filtering and group-by works on MV columns. For example:

- A predicate matches a row as long as there's one value from the MV that satisfies the predicate (give examples).
- Group-by on mv columns will flatten the groups. And what happens if there are multiple MV group by columns, etc.

File: basics/components/schema.md (hunk 1 above)
Comment (mneedham) [reply basics/components/schema.md:94]: I actually did the changes on this PR via gitbook, so I've now applied your suggestion via gitbook as well!

**Comment by mneedham** [comment]
> Thanks for adding this. It would be good to also add more information on how filtering and group-by works on MV columns. For example:

> A predicate matches a row as long as there's one value from the MV that satisfies the predicate (give examples).
> Group-by on mv columns will flatten the groups. And what happens if there are multiple MV group by columns, etc.

This is a good idea as well, although I'm not sure where would be the best place for it to go.I'll have a look through the other docs and see if I can find a good spot.

**Pull request closed without merging**